src/factory/providers/zksync-era/increment/index.ts

```diff
@@ -1,18 +1,35 @@
 import util from '../../../../util/blockchainUtil';
+import basicUtil from '../../../../util/basicUtil';
 import formatter from '../../../../util/formatter';
 import { ITvlParams, ITvlReturn } from '../../../../interfaces/ITvl';
 import VAULT_ABI from './abi.json';
 
 const START_BLOCK = 28541484;
 const VAULT_ADDRESS = '0x703500cAF3c79aF68BB3dc85A6846d1C7999C672';
 const USDC = '0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4';
+type Collateral = {
+  asset: string;
+  weight: BigInt;
+  decimals: number;
+  currentAmount: BigInt;
+  maxAmount: BigInt;
+};
```

Comment:
Given the comment above, I'm not sure this type definition does any good here

Reply:
Yeah, this would not be needed.